src/Agent/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### New Features
+* Adds the `Logging/Forwarding/Dropped` metric to track the number of log messages that were dropped due to capacity constraints. [#1470](https://github.com/newrelic/newrelic-dotnet-agent/pull/1470)
```

Comment:
This is a supportability metric so the full name would be `Supportability/Logging/Forwarding/Dropped`.  I don't recall us typically announcing supportability metrics in the CHANGELOG or release notes.

Reply:
Never mind, should have searched our docs instead of relying on my memory!  We should still correct the name of the metric.
```suggestion
* Adds the `Supportability/Logging/Forwarding/Dropped` metric to track the number of log messages that were dropped due to capacity constraints. [#1470](https://github.com/newrelic/newrelic-dotnet-agent/pull/1470)
```